src/System.Collections/tests/ILCConfigurations.rd.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
+    <Assembly Name="System.Collections" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
I will be following yo with the codegen team to get this fixed and then I will take care of removing this when that gets fixed!